res/notes/icons.txt

```diff
@@ -36,7 +36,7 @@ Font
  #                        #
  ##########################
 
-DOS 3.3 (could also just use 5.25 Floppy)
+DOS 3.3 (could also just use 5.25 floppy)
```

Comment:
Please revert changes to this file. It's inconsequential and just adds clutter to the revision history.